src/Parser/BalanceDruid/Modules/Features/CastEfficiency.js

```diff
@@ -0,0 +1,74 @@
+import ITEMS from 'common/ITEMS';
+import SPELLS from 'common/SPELLS';
+
+import CoreCastEfficiency from 'Parser/Core/Modules/CastEfficiency';
+
+class CastEfficiency extends CoreCastEfficiency {
+  static CPM_ABILITIES = [
```

Comment:
It would be nice if *all* abilities were here in one form or another. As an example look at the Disc Priest CastEfficiency which has *rotational spell*, *cooldowns*, *spells* and *utility* categories. It even has Shackle Undead so that the list is complete.

Reply:
Don't you think that's too much useless info? Druids have a lot of utility things that don't affect rotation in any way (lots of them are not even in GCD).
I can see how a innervate thing would be useful, but fail to see how, for instance, Typhoon would.

Reply:
Abilities such as Typhoon being useful in a fight is pretty rare, but showing it in the list might reveal to someone that it's a spell they have available. On a fight where such a is useful to use on cooldown, you can see the cast efficiency. Of course disabling suggestions and the *can be imporved* indicator would likely be desirable, as can be done with the available properties.

Reply:
You can take a look at AfflictionWarlock/Features/CastEfficiency, I too had this notion that it would be highly unimportant, but when you add the additional properties, the only place it's gonna stay in is the Cast Efficiency tab so I don't think that's a huge problem if it's not cluttering place anywhere else

Reply:
Hmm ok then :) will look at it tonight or tomorrow